content/04.results.md

```diff
@@ -41,3 +41,18 @@ We observed that including one label function of a mismatched type to distant su
 We found that usually adding random noise did not improve performance (Figure {@fig:auroc_random_label_function_performance} and Supplemental Figure {@fig:aupr_random_label_function_performance}).
 For the CbG edge type we did observe slightly increased performance via AUPR (Supplemental Figure {@fig:aupr_random_label_function_performance}).
 However, performance changes in general were smaller than those observed with mismatched label types.
+
+### Reconstructing Hetionet
+
+![
+Barchat that quantifies how many existing edges we could recall and novel edges that can be added to hetionet.
+Existing edges are shown in the green bar and the blue bar represents novel edges can be added.
```

Comment:
I don't entirely understand this figure from the legend. There are numbers and percentages on the figure. I need more of a walkthrough. This figure is complex, and should probably have a 2-3 sentence figure legend telling me what the elements of the figure are.